src/main/java/sc/fiji/bdvpg/sourceandconverter/importer/NewSourceAndConverterGetter.java

```diff
@@ -0,0 +1,142 @@
+package sc.fiji.bdvpg.sourceandconverter.importer;
+
+import bdv.util.RandomAccessibleIntervalSource;
+import bdv.viewer.Source;
+import bdv.viewer.SourceAndConverter;
+import net.imglib2.RandomAccessibleInterval;
+import net.imglib2.img.Img;
+import net.imglib2.img.ImgFactory;
+import net.imglib2.realtransform.AffineTransform3D;
+import net.imglib2.util.Util;
+import sc.fiji.bdvpg.sourceandconverter.SourceAndConverterUtils;
+
+import java.util.function.Supplier;
+
+public class NewSourceAndConverterGetter implements Runnable, Supplier<SourceAndConverter> {
+
+    AffineTransform3D at3D = new AffineTransform3D();
+
+    long nx, ny, nz;
+
+    String name;
+
+    ImgFactory imgfactory;
+
+    public NewSourceAndConverterGetter(
+            String name,
+            AffineTransform3D at3D,
+            long nx, long ny, long nz,
+            ImgFactory imgfactory
+            ) {
+        this.nx = nx;
+        this.ny = ny;
+        this.nz = nz;
+        this.at3D = at3D;
+        this.name = name;
+        this.imgfactory = imgfactory;
+    }
+
+    public NewSourceAndConverterGetter(
```

Comment:
Could you give this class a more specific name? This is very generic 😄 at least I cannot agues from the name what it is doing.... 

Also, is it maybe more a `Creator` or a `Factory` than a `Getter`? With `Getter` I associate that I get an object that exists already...

Reply:
Changed to `EmptySourceAndConverterCreator` in https://github.com/bigdataviewer/bigdataviewer-playground/pull/97/commits/612860d6ac98ae3016a6b4353fc0fb049afa362a